docker/Dockerfile.ubuntu

```diff
@@ -22,7 +22,13 @@ RUN apt-get -qq update && \
     apt-get -y install build-essential libelf-dev libfl-dev clang-10 libcap-dev
 
 RUN cd /root/bcc/src/cc/libbpf/src/ && make
-RUN cd /root/bcc/libbpf-tools/ && \
+# Find absolute path of BPFTOOL to avoid not finding it for obscure reasons near
+# the end of the CI.
+RUN arch=$(dpkg --print-architecture); \
+  if [ "${arch}" != 'amd64' ]; then \
+    export BPFTOOL=bin/bpftool-${arch}; \
+  fi; \
+  cd /root/bcc/libbpf-tools/ && \
   CLANG=clang-10 LLVM_STRIP=llvm-strip-10 make -j $(nproc) DESTDIR=/libbpf-tools/ prefix="" install
 
 FROM ubuntu:${OS_TAG}
```

Comment:
Is the resulting container image still based on this? If yes, unless you have an arm64 runner here it will take the runner's arch image, i.e., amd64 - you would have to force the arm64 Ubuntu image here. But maybe you can ignore me if docker buildx already does this for you^^

Reply:
Seems it is https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images, good to know

Reply:
I normally used `buildx` to deal with it and it worked when I tested it mid-december.
I just need to rebase and build an up to date arm image to test and validate everything.